### DISCUSSION TITLE
How to get the information of the user only by instance and token?

### DISCUSSION THREAD

**Discussion by muzuiyo** [neet/masto.js#1077] [Q&A] [answered]
Hi, I am looking for a way to get the information of the user without accountid or username. Is it possiable to achieve that like the code below?
```javascript
const masto = createRestAPIClient({
    url: instance,
    accessToken: token
});

const account = masto.getAccount();
console.log(account.lastStatusAt);
```

**Comment by neet** [comment, marked as answer]
Hi @maodaisuki, You may want `/api/v1/accounts/verify_credentials`. See the documentation below:
https://docs.joinmastodon.org/methods/accounts/#verify_credentials

The corresponding API for it in Masto.js is `masto.v1.accounts.verifyCredentials()`

https://github.com/neet/masto.js/blob/b354d7ab513287ed1679159f780f8f9fb3257394/tests/rest/v1/accounts.spec.ts#L77-L81

**Reply by muzuiyo** [reply]
This's exactly what I want, thank you very much!
